### DISCUSSION TITLE
OG-Core Developer Meeting, June 19, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#870] [Meeting]
**Agenda for OG-Core developer meeting: 6/19/23**
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

**Agenda**
* Adding idiosyncratic income uncertainty
* Calibration of OG-USA and calibration documentation
* July 3 meeting question

Next meeting: Monday, July 3, noon-1pm ET

@rickecon @jdebacker @jpycroft @nikhilwoodruff @SeaCelo @sarvonz @prrathi

**Comment by nikhilwoodruff** [comment]
Unfortunately won't be able to make this meeting as Max and I are in London- no major updates on the UK end.
